crucible-mir/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
-# next -- TBA
+# 0.9 -- 2024-08-30
 
-* Fix a bug in which `crucible-mir` would fail to parse MIR JSON code involving
-  casts from array references to pointers.
+* Implement byte-to-char casts for crucible-mir
```

Comment:
You may as well drop the "for crucible-mir" part, considering that this is in the `crucible-mir` changelog to being with.